src/main/java/uk/ac/ebi/ampt2d/registry/service/mail/MailService.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.registry.service.mail;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MailService {
+
+    @Autowired
+    private JavaMailSender javaMailSender;
+
+    @Value("${mail.to}")
+    private String to;
+
+    @Value("${mail.from}")
+    private String from;
+
+    @Value("${mail.subject}")
+    private String subject;
+
+    public void send(String text) {
+        SimpleMailMessage message = new SimpleMailMessage();
+        message.setTo(to);
+        message.setFrom(from);
+        message.setSubject(subject);
+        message.setText(text);
+        javaMailSender.send(message);
```

Comment:
Could you please handle any exceptions that may arise? Sending the mail could fail due to internal or external factors.

Reply:
Furthermore, is there a way to group multiple messages into a single mail. If there are loads of messages coming at the same time, I image the mail server would not be happy.

Reply:
the entity-listeners can be handled per entity basis only , grouping messages can be done only if done manually . So we can switch off notification if updates are done in bulk.